images/devcontainer/default.nix

```diff
@@ -94,23 +128,19 @@ let
       # make sure /tmp exists
       mkdir -m 0777 tmp
 
-      # allow ubuntu ELF binaries to run. VSCode copies it's own.
-      mkdir -p lib64
-      ln -s ${glibc}/lib64/ld-linux-x86-64.so.2 lib64/ld-linux-x86-64.so.2
```

Comment:
are you sure you don't want to keep this? Other extensions might also download random binaries.

Reply:
I would add later if there would be some issues, would not make this default as it solves only part of problem. If some third party binary is not statically linked, it's a good chance it will require also some other libraries. Regarding installing of vscode plugins can probably be automatized using `home-manager` module and binaries can be patched there.

Reply:
please don't remove this otherwise it will break actions/checkout@v3 when we use devcontainer as github action iamge